Simple product duplication for June updates (#617)
akeneo/pim-helpcenter: content/updates/2020-06/1-duplicate-a-product.md

# New: duplicate a product
::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
:::

> Want to easily create a new product based on another? You can now duplicate a simple product in just one click! What a time-saver! :wink:  

Reviewer: ```suggestion
Do you want to easily create a new product from another? You can now duplicate a simple product in your user interface with one single click! What a time-saver! :wink:  
```

> Do you want to easily create a new product from another? You can now duplicate a simple product in your user interface with one single click! What a time-saver! :wink:  

Reviewer 2: ```suggestion
You want to easily create a new product based on an existing one? You can now duplicate a simple product in your user interface in a single click! What a time-saver! :wink:  
```

> You can find this new action in the `...` button of the product edit form of each one of your products. 

Reviewer: ```suggestion
Find this new action under the `...` button at the top right corner of each of your product edit forms. 
```

> Find this new action under the `...` button at the top right corner of each of your product edit forms. 

Reviewer 2: ```suggestion
Find this new action under the `...` button in the top right corner of each of your product edit forms. 
```

Reviewer 2: I'm not sure about the 'each of your pef' -> would it be accurate to say in the top right corner of the pef (singular)? if It works, it'd be lighter!

Author: Yes, that'd work!

> There is a permission to hide this action, if you don't want some users to be able to duplicate products.

Reviewer: This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.

Author: ```suggestion
This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.
```

> This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.

Reviewer 2: ```suggestion
This new feature also comes with dedicated permission, so you can decide which group(s) of users can perform this action.
```
